ext/DynamicQuantitiesLinearAlgebraExt.jl

```diff
@@ -1,8 +1,69 @@
 module DynamicQuantitiesLinearAlgebraExt
 
-import LinearAlgebra: norm
-import DynamicQuantities: AbstractQuantity, ustrip, dimension, new_quantity
+import Base: (*) #, transpose
+import LinearAlgebra: norm, inv, (\), svd, Algorithm, default_svd_alg, SVD, Diagonal, Adjoint, Transpose, AbstractRotation, AbstractMatrix, eigen, eigsortby, Eigen, det
+import DynamicQuantities: AbstractQuantity, ustrip, dimension, new_quantity, AbstractDimensions, QuantityArray, Quantity
 
 norm(q::AbstractQuantity, p::Real=2) = new_quantity(typeof(q), norm(ustrip(q), p), dimension(q))
 
+# handle ambiguities
+*(q::QuantityArray,r::QuantityArray) = QuantityArray(ustrip(q)*ustrip(r),dimension(q)*dimension(r))
+*(q::QuantityArray,r::AbstractMatrix) = QuantityArray(ustrip(q)*r,dimension(q))
+*(q::QuantityArray,r::AbstractVector)  = QuantityArray(ustrip(q)*r,dimension(q))
+*(r::AbstractMatrix,q::QuantityArray) = QuantityArray(r*ustrip(q),dimension(q))
+*(r::AbstractVector,q::QuantityArray) = QuantityArray(r*ustrip(q),dimension(q))
+
+# a choice to make this function type stable (to be reviewed/revisited)
+#transpose(q::QuantityArray) = QuantityArray(transpose(ustrip(q)),dimension(q))
+
+\(q::QuantityArray,r::QuantityArray) = QuantityArray(ustrip(q)\ustrip(r),dimension(r)/dimension(q))
```

Comment:
Can limit these to only matrices by setting the second parametric type to `2` instead of *all* QuantityArray.

Reply:
(If we want)

Reply:
I attempted to restrict to 1D or 2D arrays, i.e., second parametric type to `1` or `2`. Please check code style/correctness and I can revise. Added another test to double-check.

Reply:
Looks good!